law.py

```diff
@@ -0,0 +1,47 @@
+# Copyright MaxAndMitchCorp 2020
+
+# This class' aim is to construct an object which allows for varying distributions
+# for law scores, as well as the population with which it impacts.
+import numpy as np
+from society import Society
+from person import Person
+import random
+
```

Comment:
So I am a bit of a split mind here. Here's my thoughts:

1. this is good overall design, but I feel like if I think about "what the institution does", it GENERATES Law objects. Right now, the institution is passing in parameters and then the Law class is "making itself" in a sense.

2. I think the Institution class should MAKE the Laws. I think each Law though needs to contain information about how it effects the INDIVIDUAL society members. I.E. I imagine each Law object has a list of effected person_ids, and each entry is the + or - benefit of that Law, that effects that person.

Does that make sense?

Reply:
Added vector to the law class which has a -1, 0 or 1 at the location of each person_id in person_vector which represents the detriment, null, or benefit effect to each person in the society, calculated within and by the institution.